relay.go

```diff
@@ -323,6 +328,7 @@ func (r *Relayer) Receive(f *Frame, fType frameType) (sent bool, failureReason s
 	}
 	select {
 	case r.conn.sendCh <- f:
+		sent = true
```

Comment:
Looks like this is included in #845 -- not sure if this should be rebased on top of that change?

Reply:
yea, there was a bit of a mixup when pushing this, too many parallel PRs...